lua/lspconfig/sixtyfps.lua

```diff
@@ -0,0 +1,24 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+configs.sixtyfps = {
+    default_config = {
+        cmd = {'sixtyfps-lsp'},
+        filetypes = {'sixtyfps'},
+        root_dir = function(filename) return util.path.dirname(filename) end
```

Comment:
I'd say replace this with `util.find_git_ancestor` and then add `single_file_support = true`. On the other hand, I'm not sure how SixtyFPS handles a `null` root.

 ```suggestion
        root_dir = util.find_git_ancestor,
        single_file_support = true,
```

Reply:
That was my concern too :/ I need to test